README.md

```diff
@@ -27,36 +27,44 @@ Attributes
 * `node['le']['account_key']` - your Logentries account_key (this can be found following [this link](https://logentries.com/doc/accountkey/))
 * `node['le']['hostname']` - sets the hostname of the log to the machine name, defaults to `node['hostname']`
 * `node['le']['logs_to_follow']` - An array of logs to follow or a hash of arrays
-* `node['le']['datahub']['enable']` - Defaulted to false
+* `node['le']['datahub']['enable']` - To send logs to datahub set this to true. Default is false
 * `node['le']['datahub']['server_ip']` - IP of your datahub server
 * `node['le']['datahub']['port']` - port datahub is running on, normally port 10000
-* `node['le']['pull-server-side-config']` - Specifies whether to make an api call to pull configuration or not, by default this is set to true meaning an api call will be made to th
+* `node['le']['pull-server-side-config']` - Specifies whether to make an api call to pull configuration or not, by default this is set to true meaning an api call will be made to logentries.com. Default is true
 
 ### Example of logs_to_follow
-caveats - name needs to be unique
-globbing is used in the agent -
+* caveats - name needs to be unique
+* globbing is enabled via the agent
 
 Usage
 -----
-There are 3 main scenarios in which the Logentries Linux Agent can be run
+There are 3 main scenarios in which the Logentries Linux Agent can be run.
 
-### node['le']['datahub']['enable'] = false and node['le']['pull-server-side-config'] = true
-
-* override['le']['logs_to_follow'] = ['/var/log/syslog']'
-* override['le']['logs_to_follow'] = [{:name => 'syslog', :log => '/var/log/syslog'}]
+#### Default (no datahub and pull configuration from logentries.com)
+```ruby
+override['le']['account_key'] = <logentries_account_key>
+override['le']['logs_to_follow'] = ['/var/log/syslog']'
+override['le']['logs_to_follow'] = [{:name => 'syslog', :log => '/var/log/syslog'}]
+```
 
 This is the normal scenario where you install the agent, override node['le']['account_key'] and your logs_to_follow as above.
```

Comment:
Maybe something like:

This is the normal case where you send the data directly to Logentries and get the configuration for your logs from Logentries as well.
To send data to logentries you will have to override `node['le']['account_key']`